alf/algorithms/rnd_algorithm.py

```diff
@@ -44,30 +43,64 @@ class RNDAlgorithm(Algorithm):
     def __init__(self,
                  target_net: Network,
                  predictor_net: Network,
-                 reward_adapt_speed=8.0,
+                 encoder_net: Network = None,
+                 reward_adapt_speed=None,
                  observation_adapt_speed=None,
                  observation_spec=None,
+                 learning_rate=None,
+                 clip_value=-1.0,
+                 stacked_frames=True,
                  name="RNDAlgorithm"):
         """
         Args:
+            encoder_net (Network): a shared network that encodes imgs to
```

Comment:
```encodes imgs ...```, is there an explicit assumption that the inputs have to be images? It seems not.

Reply:
If it's not, then this can be None